src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -143,6 +144,21 @@ public static Weight parseWeight(String weight) throws ParseException {
         return new Weight(trimmedWeight);
     }
 
+    /**
+     * Parses a {@code String bloodType} into a {@code BloodType}.
```

Comment:
If you refer to the javadoc for the other methods, shouldn't it be `@code String BloodType` and `@code bloodType`?